sdks/python/apache_beam/transforms/core.py

```diff
@@ -1435,18 +1429,6 @@ def expand(self, pcoll):
     self._check_pcollection(pcoll)
     return pvalue.PCollection(pcoll.pipeline)
 
-  def to_runner_api_parameter(self, context):
```

Comment:
Why was this removed?  This change conflicts with the changes in https://github.com/apache/incubator-beam/pull/4529.